lms/djangoapps/student_account/views.py

```diff
@@ -302,6 +311,45 @@ def _external_auth_intercept(request, mode):
         return external_auth_register(request)
 
 
+def _get_order_details(user):
```

Comment:
This should be a proper method not a private method so rename it to just `get_order_details`